### DISCUSSION TITLE
feat: error messages positions

### DISCUSSION THREAD

**Discussion by Pyakz** [mantinedev/mantine#6514] [Q&A]
![Screenshot 2024-07-12 at 3 43 32 PM](https://github.com/user-attachments/assets/ff797177-15ed-4ba7-843c-190576d0e5d7)

This is my solution for now, but its very redundant

![Screenshot 2024-07-12 at 3 45 58 PM](https://github.com/user-attachments/assets/ba9714fd-7eb6-4630-9e2c-86de2738ddff)

**Comment by rtivital** [comment]
https://mantine.dev/core/input/#default-props-on-theme
https://mantine.dev/core/input/#styles-on-theme
